### PR TITLE
Add compare coverage

### PR DESCRIPTION
**Pull request by BetsyMcPhail**
Two outstanding issues:

1. Filters
2. Sorting on the percentage columns

PTAL @zackgalbreath

### CODE REVIEW

**Comment by zackgalbreath** [comment]
@BetsyMcPhail could you revisit this topic in light of #250?  In particular, it should now be possible to simplify the logic in ```api/v1/compareCoverages.php```.

File: public/api/v1/compareCoverage.php [hunk 1]
```diff
@@ -0,0 +1,668 @@
+<?php
+/*=========================================================================
```
Comment (jamiesnape) [inline public/api/v1/compareCoverage.php]: Replace with new license header.

File: public/compareCoverage.php [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?php
```
Comment (jamiesnape) [inline public/compareCoverage.php]: Add license header.

**Comment by BetsyMcPhail** [comment]
@zackgalbreath PTAL.

**Comment by zackgalbreath** [comment]
@BetsyMcPhail As we discussed, I pushed a commit to simplify the code in api/v1/compareCoverage.php.  Let me know if you see any problems with this approach.

**Comment by BetsyMcPhail** [comment]
That looks much better @zackgalbreath .

**Pull request merged**